plugin/modelgen/models_test.go

```diff
@@ -65,9 +65,9 @@ func TestModelGeneration(t *testing.T) {
 
 		expectedTags := []string{
 			`json:"missing2" database:"MissingTypeNotNullmissing2"`,
-			`json:"name" database:"MissingInputname"`,
-			`json:"missing2" database:"MissingTypeNullablemissing2"`,
-			`json:"name" database:"TypeWithDescriptionname"`,
+			`json:"name, omitempty" database:"MissingInputname"`,
+			`json:"missing2, omitempty" database:"MissingTypeNullablemissing2"`,
+			`json:"name, omitempty" database:"TypeWithDescriptionname"`,
```

Comment:
```suggestion
			`json:"name,omitempty" database:"MissingInputname"`,
			`json:"missing2,omitempty" database:"MissingTypeNullablemissing2"`,
			`json:"name,omitempty" database:"TypeWithDescriptionname"`,
```

Reply:
In line 84-89, change it to this:
```
		expectedTags := []string{
			`json:"name,omitempty" anotherTag:"tag"`,
			`json:"enum,omitempty" yetAnotherTag:"12"`,
			`json:"noVal,omitempty" yaml:"noVal" repeated:"true"`,
			`json:"repeated,omitempty" someTag:"value" repeated:"true"`,
		}
```

After both of these changes, running:
```
go generate; cd _examples; go generate; cd ..
go test ./...; cd _examples; go test ./...
```

All test will pass.